src/test/java/com/parrit/services/PairingServiceTest.java

```diff
@@ -225,12 +227,15 @@ public void getSortedPairingHistory_getsThePairingHistoriesForAProject_sortedByM
                 new PairingHistory(project, "Pairing Board 2", new ArrayList<>(), new Timestamp(50))
         );
 
-        when(mockPairingHistoryRepository.findByProjectOrderByTimestampDesc(any(Project.class))).thenReturn(pairingHistories);
+        when(mockPairingHistoryRepository.findByProjectAndTimestampAfter(any(Project.class), any(Timestamp.class))).thenReturn(pairingHistories);
+        Timestamp mockNow = Timestamp.valueOf("2020-06-30 00:00:00.000000000");
+        when(mockCurrentTimeProvider.getCurrentTime()).thenReturn(mockNow);
 
         List<PairingHistory> result = pairingService.getSortedPairingHistory(project);
 
         assertThat(result, equalTo(pairingHistories));
 
-        verify(mockPairingHistoryRepository).findByProjectOrderByTimestampDesc(project);
+        Timestamp thirtyDaysAgo = Timestamp.valueOf("2020-05-31 00:00:00.000000000");
+        verify(mockPairingHistoryRepository).findByProjectAndTimestampAfter(project, thirtyDaysAgo);
```

Comment:
I should add test for multiple pair histories and the order in which they are returned.